test/resources/unittests.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Are these just random certs to see if the parser/IO works? They don't have any permissions anywhere right?

Reply:
I copied them from aws-c-io's unit tests.
The unit tests I have so far are just testing the parser,
but when I get around to writing tests that make actual connections, we'll use them in similar ways to how they're used in aws-c-io

Reply:
I assume they have no permissions, since they're publicly in aws-c-io